packages/trilinoscouplings/examples/scaling/example_Maxwell.cpp

```diff
@@ -403,7 +406,16 @@ int evalCurlCurlu(double & curlcurlu0,
 /**********************************************************************************/
 /******************************** MAIN ********************************************/
 /**********************************************************************************/
+int body(int argc, char *argv[]);
+
 int main(int argc, char *argv[]) {
+  Kokkos::initialize(argc,argv);
```

Comment:
If you use MPI, then you should initialize kokkos immediately after initializing MPI. Either move the GlobalMpiSession here or move the kokkos::init into body and call after constructing the mpi session object. This is needed so that kokkos gets the process binding masks from MPI:

https://kokkos.github.io/kokkos-core-wiki/ProgrammingGuide/Initialization.html#initialization

Reply:
Fixed for this test and the source of my pattern match change ;-)